schema.graphql

```diff
@@ -95,63 +69,28 @@ type Market @entity {
   name: String!
   "The Protocol the Market is a part of"
   protocol: Protocol!
-  "All Accounts that have used this Market"
-  accounts: [AccountInMarket!]! @derivedFrom(field: "market")
-  "Asset deposited in the market as collateral"
-  depositAsset: Asset! # NOTE - this could be an array, but we should find a protocol that needs this first, before implementing like so
-  "Asset borrowed. CDP style is native asset - i.e. DAI. Aave style borrowed asset == deposit Asset"
-  borrowAsset: Asset! # NOTE - this could be an array, but we should find a protocol that needs this first, before implementing like so
+  "Underlying Asset"
+  asset: Asset!
+  "Underlying token symbol"
+  underlyingSymbol: String!
+  "Underlying token address"
+  underlyingAddress: Bytes!
+  "Underlying token name"
+  underlyingName: String!
```

Comment:
We can just get these values from the `asset: Asset!` above.
such as:
```js
market.asset.symbol
market.asset.name
```

so these 3 can be removed